platform/stub-setup.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Injects the stubs into the global scope.
+ * Bundled HomeApp cannot be loaded until these globals are set.
+ */
+import { DeviceManagerStub } from './device-manager';
+import { AppStub } from './smart-home-app';
+
+(global as any).smarthome = {
+  App: AppStub,
+  DeviceManager: DeviceManagerStub,
+};
+
+//TODO(cjdaly) find a cleaner way to do this.
+export function loadHomeApp(path: string) {
```

Comment:
I'm not sure if you need this now, if it's being used as a library.

Reply:
Will address in another PR